packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -873,6 +873,7 @@ export class DashboardModel {
                 `${DashboardVersionsTableName}.dashboard_id`,
                 `${DashboardVersionsTableName}.created_at`,
             )
+            .distinctOn(`${DashboardVersionsTableName}.dashboard_id`)
```

Comment:
why is this distinct needed if you're doing .table(DashboardsTableName) and innerJoin is with projects ?

Reply:
because is joining the dashboard version table